tests/subsys/suit/tests/sdfw/Kconfig

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config SUPPORT_SUIT_STORAGE_LAYOUT_TEST
```

Comment:
I have a mixed feelings about putting this file inside `sdfw` directory, but I do not have a better idea to propose.

Interestingly, this file configures storage symbol, but does not MPI nor validator. This is not necessary today, but it justifies the generic naming scheme of this file.